packages/cli/src/commands/db.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable eqeqeq */
```

Comment:
it seems `==` is being used intentionally here over `===`, so I'm disabling this rule for this file only. In general, I think this is a rule we want to promote, so I didn't disable it globally

Reply:
Hmm. Exit code is a number so to me that looks like it should be referential equality.

Reply:
`code` can actually be `null`:
https://nodejs.org/api/child_process.html#child_process_event_exit

> If the process exited, code is the final exit code of the process, otherwise null

Looking at this,`===` would definitely be what we want here, since we are targeting the success case. I'll remove the disable and fix the equality comparison

Reply:
Typescript is also inferring it properly as `number | null` so I'm removing the type declaration on the argument of the exit event

Reply:
I love it when linting solved real world bugs! :)